identity/src/main/java/com/stripe/android/identity/ui/OTPScreen.kt

```diff
@@ -66,6 +69,11 @@ internal fun OTPScreen(
         val otpViewModel: OTPViewModel = viewModel(
             factory = otpViewModelFactory
         )
+
+        LaunchedEffect(Unit) {
+            otpViewModel.initialize()
+        }
```

Comment:
Why did this become necessary as opposed to just using the ViewModel’s `init` block?

Reply:
because we might go back to this screen later, which would require us to trigger the `initialize` call again. `init` block won't trigger again